templates/arc/app/entry.server.tsx

```diff
@@ -24,7 +24,7 @@ export default function handleRequest(
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
   loadContext: AppLoadContext
 ) {
-  return isbot(request.headers.get("user-agent"))
+  return isbot(request.headers.get("user-agent") || "")
```

Comment:
Templates use v4 API moving forward

Reply:
The use case has been addressed with version 4.2.0 to accept "null" as input, with the intention to keep user code cleaner.

https://github.com/omrilotan/isbot/blob/main/CHANGELOG.md#420
> Accept null in place of user agent string to allow header value to be used "as is" (request.headers.get("user-agent"))